examples_tests/60.ClusteredRendering/cull_common.glsl

```diff
@@ -0,0 +1,247 @@
+#ifndef CULL_COMMON_H
+#define CULL_COMMON_H
+
+#ifndef _NBL_GLSL_WORKGROUP_SIZE_
+#error "_NBL_GLSL_WORKGROUP_SIZE_ must be defined"
+#endif
+
+#ifndef LIGHT_CONTRIBUTION_THRESHOLD
+#error "LIGHT_CONTRIBUTION_THRESHOLD must be defined"
+#endif
+
+#ifndef LIGHT_RADIUS
+#error "LIGHT_RADIUS must be defined"
+#endif
+
+#define INVOCATIONS_PER_LIGHT 8
+#define LIGHTS_PER_WORKGROUP (_NBL_GLSL_WORKGROUP_SIZE_/INVOCATIONS_PER_LIGHT)
+
+layout(local_size_x = _NBL_GLSL_WORKGROUP_SIZE_) in;
+
+#include <nbl/builtin/glsl/shapes/aabb.glsl>
+#include <nbl/builtin/glsl/math/quaternions.glsl>
+
+#include <../intersection_record.glsl>
+
+struct nbl_glsl_ext_ClusteredLighting_SpotLight
+{
+	vec3 position;
+	float outerCosineOverCosineRange;
+	uvec2 intensity;
+	uvec2 direction;
+};
```

Comment:
can call it `nbl_glsl_BasicSpotLight` and put it in `nbl/builtin/glsl/lights/basic_spot_light.glsl`

Reply:
make sure to comment on the formatting of the members and what they hold